src/App.css

```diff
@@ -23,3 +23,8 @@
 .App-link {
   color: #61dafb;
 }
+
+.body {
+  background-color: #fafafa;
+  font-family: Arial, sans-serif;
+}
```

Comment:
- [ ] Please ensure the layouts fit the screen size. The page should not be scrolling horizontally. Let's work on that to improve the UI design.